manifests/providers/params.pp

```diff
@@ -9,7 +9,7 @@
   # OS specific package names
   case $::osfamily {
     'RedHat': {
-      if versioncmp($::puppetversion, '4.0') >= 0 {
+      if versioncmp($::puppetversion, '4.0') >= 0 and $::rubysitedir =~ /\/opt\/puppetlabs\/puppet/ {
```

Comment:
One reason to leave it out is to potentially support old PE installs, but since nobody has asked for that I think the only real reason is simplicity.